### PR TITLE
#169258621 Integrate Travis CI with readme badge

### PR DESCRIPTION
**Pull request by WillyBoris**
#### What does this PR do?
Integrate Travis CI with readme badge
#### Description of Task to be completed?
- Add .travis.yml and its configuration
- Add Travis badge to the readme
#### How should this be manually tested?
- You have to clone the repo `git clone https://github.com/andela/team-odd-bn-backend.git`
- Then After the repo cloned `cd project-folder` and run `npm i` to install all dependencies
- Go to my branch `git checkout ch-integrate-travis-CI-with-readme-badge-169258621`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#169258621](https://www.pivotaltracker.com/story/show/169258621)
#### Screenshots (if appropriate)
image

#### Questions:
N/A

### CODE REVIEW

**Comment by nziokaivy** [comment]
@WillyBoris Why is TravisCI is failing?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - 'stable'
+install: npm install
+services:
+  - postgresql
+before_script:
+  - psql -c 'create database barefoot_test;' -U postgres
+script: npm test
```
Comment (victkarangwa) [inline .travis.yml:9]: Good job @WillyBoris for the progress.
As a suggestion for seamless work, we can add configurations that will be used to trigger coveralls by the time we are working on coverage:
```
 after_success:
  - npm run coveralls
```

**Review by victkarangwa — APPROVED** [review]

**Comment by victkarangwa** [comment]
> @WillyBoris Why is TravisCI is failing?

@nziokaivy I think we don't have any test for travisCI to build on the top of it.

File: .travis.yml (hunk 1 above)
Comment (WillyBoris) [reply .travis.yml:9]: yeah, it's very needed `npm run coverage`. Thanks

**Review by nziokaivy — APPROVED** [review]
Great Job Willy! I think this is good to go!

**Comment by nziokaivy** [comment]
@WillyBoris Probably just the resolve the conflict

**Review by hezronkimutai — APPROVED** [review]
Great work willy, I think we are good to go. just resolve the conflict.

**Comment by julietezekwe** [comment]
Please fix the hound violations and I will merge. @WillyBoris

**Comment by WillyBoris** [comment]
> Please fix the hound violations and I will merge. @WillyBoris

>Yes, I corrected it, but let me know if the way I did, is right.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - 'stable'
+install: npm install
+services:
+  - postgresql
+before_script:
+  - psql -c 'create database barefoot_test;' -U postgres
```
Comment (NiyoEric) [inline .travis.yml]: @WillyBoris the test database should be `barefoot_nomad_test_db`

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+script: npm test
+after_success: 
+ - npm run coverage
```
Comment (NiyoEric) [inline .travis.yml]: @WillyBoris you forget to add blank space at the end of line

**Review by NiyoEric — APPROVED** [review]

File: .travis.yml (hunk 3 above)
Comment (NiyoEric) [inline .travis.yml]: @WillyBoris you forget to add blank space at the end of line

**Review by julietezekwe — APPROVED** [review]

**Pull request merged**